cmd/minikube/cmd/start.go

```diff
@@ -577,7 +577,7 @@ func validateSpecifiedDriver(existing *config.ClusterConfig) {
 
 	out.ErrT(out.Workaround, `To proceed, either:
 
-1) Delete the existing "{{.name}}" cluster using: '{{.command}} delete'
+1) Delete the existing "{{.name}}" cluster using: 'minikube delete -p {{.name}}'
```

Comment:
For consistency, use the output of:`mustload.ExampleCmd(existing.Name, "delete")

Reply:
nice, I didn't realize that package existed. done.